README.md

```diff
@@ -1,25 +1,26 @@
 
-HydroShare is a collaborative website being developed for better access to data and models in the hydrologic sciences. HydroShare will provide the sustainable technology infrastructure needed to address critical issues related to water quantity, quality, accessibility, and management. HydroShare will expand the data sharing capability of the CUAHSI Hydrologic Information System by broadening the classes of data accommodated, expanding capability to include the sharing of models and model components, and taking advantage of emerging social media functionality to enhance information about and collaboration around hydrologic data and models. 
+HydroShare is a website and hydrologic information system for sharing hydrologic data and models aimed at giving hydrologists the cyberinfrastructure needed to innovate and collaborate in research to solve water problems. HydroShare is designed to advance hydrologic science by enabling the scientific community to more easily and freely share products resulting from their research, not just the scientific publication summarizing a study, but also the data and models used to create the scientific publication. With HydroShare users can: (1) share data and models with colleagues; (2) manage who has access to shared content; (3) share, access, visualize and manipulate a broad set of hydrologic data types and models; (4) use the web services API to program automated and client access; (5) publish data and models to meet the requirements of research project data management plans; (6) discover and access data and models published by others; and (7) use web apps to visualize, analyze, and run models on data in HydroShare.
 
 More information can be found in our [Wiki Pages](https://github.com/hydroshare/hydroshare/wiki)
 
 ## Install
 
 This README file is for developers interested in working on the Hydroshare code itself, or for developers or researchers learning about how the application works at a deeper level. If you simply want to _use_ the application, go to http://hydroshare.org and register an account.
 
-If you want to install and run the source code of application locally, read on.
+If you want to install and run the source code of application locally and/or contrib to development, read on.
 
-#### Dependencies
-[VirtualBox](https://www.virtualbox.org/wiki/Downloads) is required, as the preferred development environment is encapuslated within a VM. This VM has the appropriate version of Ubuntu  nstalled, as well as python and docker and other necessary development tools. 
+#### Depencency: [VirtualBox](https://www.virtualbox.org/wiki/Downloads) development environment
+
```

Comment:
Part of my suggestion was to delete Dependency, as Virtual Box is not really a dependency, but it is offered as an easy to use development environment.  Also I would use a header level 3 same as simplified installation instructions.